libpod/container_internal_linux.go

```diff
@@ -968,6 +968,7 @@ func (c *Container) mountNotifySocket(g generate.Generator) error {
 // systemd expects to have /run, /run/lock and /tmp on tmpfs
 // It also expects to be able to write to /sys/fs/cgroup/systemd and /var/log/journal
 func (c *Container) setupSystemd(mounts []spec.Mount, g generate.Generator) error {
+	g.AddProcessEnv("container_uuid", c.ID()[:32])
```

Comment:
I think you should check that `container_uuid` is not already set

Reply:
Fixed.